tests/test_event_dispatcher.py

```diff
@@ -44,10 +44,10 @@ def test_dispatch_event__post_request(self):
 
     url = 'https://www.optimizely.com'
     params = {
-      'accountId': '111001',
-      'eventName': 'test_event',
-      'eventEntityId': '111028',
-      'visitorId': 'oeutest_user'
+      'a': '111001',
```

Comment:
These testcases only check that event is dispatched with exactly the same params. Actual params do not make a difference.

Reply:
I fail to understand the need to do this. Why are we changing things for the sake of changing things? This is beyond the scope of the change and contributes to the review size.

Reply:
it was giving some misunderstanding like we are using eventName, eventEntityId, just for the sake of removing any misconception, we just renamed it.

Reply:
Please undo the renaming.

Reply:
ok, we will revert it.